topsort-api-v2.yml

```diff
@@ -319,20 +319,21 @@ components:
           description: The search string provided by a user.
         device:
           $ref: '#/components/schemas/Device'
-        aspectRatio:
+        slotId:
           type: string
-          pattern: '^\d+:\d+$'
           description: >
-            The aspect ratio for a specific slot for a banner placement in the marketplace app.
-            The aspect ratio for the category (or home page, if no category ID was provided in the request) must have been previously set up in the Topsort app.
+            The ID of the region for which this auction will be run for.
+            It must match the values provided when creating the banner campaign.
```

Comment:
Not sure if we should have a comment like this. Also I'm not even sure this valu is safe to be an id, as according to https://stackoverflow.com/questions/1077084/what-characters-are-allowed-in-dom-ids it must start with a letter.

Reply:
I agree with both, I ended up using banner placement.